metrics/deploy/docs/diagrams/doctor_command_sequence_diagram.puml

```diff
@@ -0,0 +1,57 @@
+@startuml doctor_command_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+box Doctor Tool
```

Comment:
It seems like the `Doctor` is not a separate tool - it is a command of the CLI. The same for the `Deploy Tool`.

```suggestion
box Doctor
```
or 
```suggestion
box Doctor Command
```